src/components/ExplainSideColumn.tsx

```diff
@@ -11,6 +11,23 @@ interface MyProps {
   backUrl: string,
 }
 
+// From here:
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value
+// because I get this error
+// https://stackoverflow.com/questions/50965551/typeerror-cyclic-object-value-in-react
+const getCircularReplacer = () => {
+  const seen = new WeakSet();
+  return (key:any, value:any) => {
+    if (typeof value === "object" && value !== null) {
+      if (seen.has(value)) {
+        return;
+      }
+      seen.add(value);
+    }
+    return value;
+  };
+};
+
```

Comment:
I think this is because you are currently saving the selected organisation in the local storage as a string and then later on, you try to stringify this string again and it caused a circle. I think it could be solved by not storing the selected organisation in local storage as JSON string but as an object.

Reply:
I just checked and it's not possible to store an object in local storage. So it means you need to parse it to an object first before using it.

Reply:
It is needed for the line below that uses a jsx component as key:
key={JSON.stringify(explanationText, getCircularReplacer)}
We could also adapt the JSX element manually to a different non circular object.
Anyway I suspect that introduces new surface for bugs, so I prefer this patch.